lib/renderers.php

```diff
@@ -1,5 +1,18 @@
 <?php
-
+/**
+ * Render a simplified UI tag for series podcasts.
+ *
+ * @param string $label The text to display inside the tag.
+ * @param string $url The link to wrap the tag in.
+ * @param string $variant_class Optional extra classes can be added for styling variants.
+ */
+function render_series_ui_tag( $label, $url, $variant_class = '' ) {
```

Comment:
There isnt really anything about this that is specific to being a series tag, so the function could be named more general for potential use in other locations